CMakeLists.txt

```diff
@@ -255,12 +255,33 @@ endif()
 set(outputFilterRegex "addons/xbmc.json")
 find_addon_xml_in_files(${outputFilterRegex})
 
+if(PIPEWIRE_FOUND)
+  list(APPEND AUDIO_BACKENDS_LIST "pipewire")
+endif()
+
+if(ALSA_FOUND)
+  list(APPEND AUDIO_BACKENDS_LIST "alsa")
+endif()
+
+if(PULSEAUDIO_FOUND)
+  list(APPEND AUDIO_BACKENDS_LIST "pulseaudio")
+endif()
+
+if(SNDIO_FOUND)
+  list(APPEND AUDIO_BACKENDS_LIST "sndio")
+endif()
+
+if(ALSA_FOUND AND PULSEAUDIO_FOUND)
+  list(APPEND AUDIO_BACKENDS_LIST "alsa+pulseaudio")
+endif()
+
```

Comment:
is it better to append to this list in the respective find module? everything except the last one is pretty straight forward. 

For scope, you'll just need to init the list in the top CMakeLists.txt for ease